test/stripe/payment_methods/bank_account_test.exs

```diff
@@ -4,14 +4,14 @@ defmodule Stripe.BankAccountTest do
   describe "update/2" do
     test "updates a bank account" do
       assert {:ok, _} = Stripe.BankAccount.update("cus_123", "ba_123")
-      assert_stripe_requested(:post, "/v1/customers/cus_123/sources/ba_123")
+      assert_stripe_requested(:post, "/v1/accounts/cus_123/external_accounts/ba_123")
```

Comment:
is this correct? or are we missing something in the map of function mapping?

Reply:
This is not correct.  There are mappings for it, but the BankAccount can is an `anyOf` that can be a `Customer.Source` or `Account.ExternalAccount. '  Since the two endpoints map to the same function name, there is a collision, and the last one appears to win in the generation.

Any direction on how you'd like to handle this?

Reply:
This also affects the `Card` module since it takes `BankAccount`

Reply:
This is the type of thing that I wanted to prevent by manually maintaining that manual mappping, some of the functions needs to be kept as they were, while the new ones, we add a new name for it.

The issue is that the order of which they appear matters, since there could be multiple `update`.

I forgot where, but there was a uniq_by somewhere that if you remove it, you would see the duplicates that I tried to prevent by manual grunk work until we were able to properly generate latest version.

Then figure out step two.

Reply:
ok, I see if I can find the mappings for BankAccount and update the function names for the external account to see if that helps.

Reply:
The mappings don't help for polymorphic resources like BankAccount.  So, I removed the ExternalAccount references in the openapi spec to make BankAccount and Card not include breaking changes.

Reply:
What do you mean by "removing" in the openapi spec?